selfdrive/car/body/values.py

```diff
@@ -26,6 +26,9 @@ class CAR:
     (Ecu.engine, 0x720, None): [
       b'02/27/2022'
     ],
+    (Ecu.debug, 0x721, None): [
+      b'166bd860' # git hash of the firmware used
```

Comment:
Do we need to specify any versions for it to work? Or can we leave an empty list?